docs/syntax.md

```diff
@@ -139,7 +137,10 @@ const InfiniteMeasure = TSXAir(() => {
 ```
 
 ### beforeUnmount
+
 ```tsx
+// @omry, this is an example of why you want to return a dispose function from afterMount
```

Comment:
Agreed. Can you think of a use case for  just beforeUnmount?

Reply:
no, not realy